index.js

```diff
@@ -25,6 +26,71 @@ module.exports = {
     ...readdirSync(join(__dirname, 'rules')).map(rule => join(__dirname, 'rules', rule))
   ],
   overrides: [
+    {
+      files: ['*.js', '*.ts'],
+      rules: {
+        'node/no-unsupported-features/es-syntax': [
+          "off"
+        ]
+      }
+    },
+    {
+      files: [
+        '*.ts'
+      ],
+      parser: '@typescript-eslint/parser',
+      // Rules inherited from https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended.ts
+      rules: {
+        'constructor-super': 'off', // ts(2335) & ts(2377)
```

Comment:
As this would be done to quash lint noise, let's keep it in specific, per-project files during the migration. Anything placed in a common component (even temporarily) creates the impression of the practice being supported and recommended.

Having the file inlined within each project makes more engineers aware of what they are missing out on. One common file per target service works, separate files per folder are even better to encourage migration.

Reply:
If I understand correctly, you're saying that turning these rules off is not the right practice, and so it's better to have these in individual `eslintrc`'s and not extended via this common module.

We can also remove these rules altogether. The purpose of these `off` rules is to de-duplicate the error from tsc as well as eslint. I'd say that this is preferable over making `eslints` everywhere more verbose.

Reply:
Reference for the de-duplication https://github.com/typescript-eslint/typescript-eslint/issues/1273

Reply:
Alright, this should go as a comment to avoid confusion for future readers :)

Reply:
Right, then adding a comment and keeping the rules.